### PR TITLE
Rnog eventbrowser

### PR DESCRIPTION
**Pull request by shallmann**
Contains modules for rno-g data i/o and improvements to the eventbrowser (w/ possibility to read rno-g files)

We would like to merge this PR since other PRs already rely on the uproot reading capabilities. We want to cleanup of the rnogDataReader and the readRNOGData scripts (similar functionality as in eventReader and NuRadioRecoio) after this PR.

### CODE REVIEW

File: NuRadioReco/detector/detector_browser/index.py
Comment (anelles) [inline NuRadioReco/detector/detector_browser/index.py:39]: Are you sure that you want the default to be in debug mode?

File: NuRadioReco/eventbrowser/apps/simulation.py
Comment (anelles) [inline NuRadioReco/eventbrowser/apps/simulation.py:9]: Same question as above. Should this really be the default?

File: NuRadioReco/eventbrowser/apps/simulation_plots/sim_event_overview.py
Comment (anelles) [inline NuRadioReco/eventbrowser/apps/simulation_plots/sim_event_overview.py:38]: Is this ARIANNA related or is this a very unfortunate remainder? 
Can also be cleaned up later, if this is harder to fix.

File: NuRadioReco/eventbrowser/apps/trace_plots/multi_channel_plot.py
Comment (anelles) [inline NuRadioReco/eventbrowser/apps/trace_plots/multi_channel_plot.py:81]: Should this be removed?

File: NuRadioReco/eventbrowser/dataprovider_root.py
Comment (anelles) [inline NuRadioReco/eventbrowser/dataprovider_root.py:15]: Somehow this seems like this is mixing RNO-G and ARIANNA now ... not sure I completely understand.

File: NuRadioReco/eventbrowser/index.py
Comment (anelles) [inline NuRadioReco/eventbrowser/index.py:232]: Here (and further below) there are a bunch on uncommented lines. Are they here for a reason (work in progress) or should they rather go?

**Review by anelles — DISMISSED** [review]
Very nice work. Thanks! I left a couple of cosmetic comments of what I noticed. I did not test this in detail, as the PR said that this is work in progress anyway and needs to be merged fast. 
Please take a look at the ARI vs RNO-G comments. If this ends up in a released, I think we want this cleaned up rather sooner than later to avoid confusion.

File: NuRadioReco/eventbrowser/dataprovider_root.py
Comment (sjoerd-bouma) [reply NuRadioReco/eventbrowser/dataprovider_root.py:15]: We inherited this from the previous eventbrowser - the file access function was called get_arianna_io. It should be relatively straightforward to rename it to something more neutral everywhere, e.g. get_io_handler

Comment (anelles) [reply NuRadioReco/eventbrowser/dataprovider_root.py]: Thanks for fixing this. This has actually been around for way too long. The files were called .ari, before we decieded for something more general as .nur

**Review by anelles — DISMISSED** [review]
I did not test it on files .. but code looks fine from my side now.

**Review by shallmann — COMMENTED** [review]
minor cleanup. Thanks @sjoerd-bouma for taking over remaining modifications!

**Comment by shallmann** [comment]
Sorry @anelles, you have to approve again, I dismissed your approval because of some minor final cosmetic changes (`import XX as XX` -> `import XX`)

File: NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py
Comment (lpyras) [inline NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py:12]: Is it correct that no logger.setLevel(logging.WARNING) or something similar is set? Maybe you can add a line here if it is not inherited.

File: NuRadioReco/eventbrowser/apps/overview_plots/event_overview.py
Comment (lpyras) [inline NuRadioReco/eventbrowser/apps/overview_plots/event_overview.py:16]: maybe set a logging level here.

File: NuRadioReco/eventbrowser/apps/overview_plots/trigger_properties.py
Comment (lpyras) [inline NuRadioReco/eventbrowser/apps/overview_plots/trigger_properties.py:43]: I am bad in html. Is that the correct format?

File: NuRadioReco/eventbrowser/apps/simulation.py
Comment (lpyras) [inline NuRadioReco/eventbrowser/apps/simulation.py:10]: nice :)

File: NuRadioReco/eventbrowser/apps/simulation_plots/sim_event_overview.py
Comment (lpyras) [inline NuRadioReco/eventbrowser/apps/simulation_plots/sim_event_overview.py:15]: set level?

**Review by lpyras — COMMENTED** [review]
Thanks for doing this!

**Review by anelles — APPROVED** [review]
Also small changes don't go unnoticed ;)

File: NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py
Comment (sjoerd-bouma) [reply NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py:12]: Loggers are created with level [logging.WARNING by default](https://docs.python.org/3/howto/logging.html#when-to-use-logging). I guess I can make all of them inherit from the NuRadioReco logger in index.py (which is currently set to logging.INFO), though.

Comment (lpyras) [reply NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py:12]: Thank you for the clarification :)

File: NuRadioReco/eventbrowser/apps/overview_plots/trigger_properties.py
Comment (shallmann) [reply NuRadioReco/eventbrowser/apps/overview_plots/trigger_properties.py:43]: yes, looks good

**Comment by shallmann** [comment]
good you spotted that hudl-mistake @sjoerd-bouma

**Review by lpyras — APPROVED** [review]
Thanks!

**Comment by shallmann** [comment]
We will keep the branch for now, in case further development for the monitoring tool is needed.

**Pull request merged**
